### PR TITLE
[Feature] [SwaggerV2] Body Parameter affect content-type header and implement formData type

### PR DESCRIPTION
**Pull request by beardeddragon5**
Swagger parameter that write to body affect the 'Content-Type' header send by the client. Insomnia didn't add this header. 

Insomnia did only support json body parameters. I added formData (x-www-form-urlencoded, multipart/form-data) support and generation of examples.

### CODE REVIEW

File: packages/insomnia-importers/src/importers/swagger2.js [hunk 1]
```diff
@@ -170,6 +172,16 @@ function importRequest(endpointSchema, globalMimeTypes, id, parentId) {
     headers: prepareHeaders(endpointSchema),
     parameters: prepareQueryParams(endpointSchema),
   };
+  if (request.body.mimeType && !request.headers.find(header => header.name === 'Content-Type')) {
```
Comment (gschier) [inline packages/insomnia-importers/src/importers/swagger2.js]: Nice! I love this solution

File: packages/insomnia-importers/src/importers/swagger2.js [hunk 2]
```diff
@@ -224,25 +236,36 @@ function prepareBody(endpointSchema, globalMimeTypes) {
...
-
   if (supportedMimeType === MIMETYPE_JSON) {
     const isSendInBody = p => p.in === 'body';
     const parameters = endpointSchema.parameters || [];
     const bodyParameter = parameters.find(isSendInBody);
     if (!bodyParameter) {
-      return {
-        mimeType: supportedMimeType,
-      };
+      return {};
```
Comment (gschier) [inline packages/insomnia-importers/src/importers/swagger2.js]: Nice cleanup here 💯

File: packages/insomnia-importers/src/__tests__/fixtures/swagger2/petstore-output.json [hunk 3]
```diff
@@ -150,7 +168,13 @@
...
       "method": "POST",
       "parameters": [],
-      "headers": [],
+      "headers": [
+        {
+          "name": "Content-Type",
+          "disabled": false,
+          "value": "multipart/form-data"
+        }
+      ],
```
Comment (gschier) [inline packages/insomnia-importers/src/__tests__/fixtures/swagger2/petstore-output.json]: Should this also have added parameters to the `body` attribute of the request? I don't see any request bodies for this change reflected in the tests.

Comment (beardeddragon5) [reply packages/insomnia-importers/src/__tests__/fixtures/swagger2/petstore-output.json]: Could be, as i pointed out in the other pull requests. The tests always returned passed. I might missed this while writing the fixtures.

**Comment by beardeddragon5** [comment]
I updated the fixtures

**Pull request merged**